### PR TITLE
Added SymEngine's license

### PR DESCRIPTION
**Pull request by Sumith1896**
cc @certik @isuruf

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+--------------------------------------------------------------------------------
+
+All files in src/teuchos were taken from the Trilinos package and are licensed
```
Comment (isuruf) [inline LICENSE]: We don't need these here

Comment (Sumith1896) [inline LICENSE]: Agreed.
Should I even remove the next part (scipy) ?

Comment (isuruf) [inline LICENSE]: Yes

Comment (Sumith1896) [inline LICENSE]: Will do

**Comment by Sumith1896** [comment]
@isuruf Updated

**Comment by isuruf** [comment]
Thanks. +1 to merge

**Comment by Sumith1896** [comment]
Thanks for the merge!

**Pull request merged**